tests/upgrades/test_client.py

```diff
@@ -254,53 +196,33 @@ def test_post_scenario_postclient_package_installation(
                 the content host is created
             3. The package is installed on post-upgrade client
         """
-        rhel7_client = dockerize(
-            ak_name=module_ak.name, distro='rhel7', org_label=default_org.label
-        )
-        client_container_id = list(rhel7_client.values())[0]
-        client_name = list(rhel7_client.keys())[0].lower()
-        # Wait 60 seconds as script in /tmp takes up to 2 min inc the repo sync time
-        time.sleep(60)
-        # Use yum install again in case script was not yet finished
-        installed_package = execute(
-            docker_execute_command,
-            client_container_id,
-            'yum -y install katello-agent',
-            host=docker_vm,
-        )[docker_vm]
-        # Assert gofer was installed after yum completes
-        installed_package = execute(
-            docker_execute_command,
-            client_container_id,
-            'rpm -q gofer',
-            host=docker_vm,
-        )[docker_vm]
-        assert 'package gofer is not installed' not in installed_package
+        rhel_client = katello_agent_client['client']
+        rhid = rhel_client.execute('subscription-manager identity')
+        assert module_org.name in rhid.stdout
+        result = rhel_client.execute('rpm -qa | grep gofer')
+        assert result.status == 0
+        assert 'package gofer is not installed' not in result.stdout
         # Run goferd on client as its docker container
-        kwargs = {'async': True, 'host': docker_vm}
-        execute(docker_execute_command, client_container_id, 'goferd -f', **kwargs)
+        rhel_client.execute("goferd -f")
```

Comment:
Same as above!

Reply:
I think this is not even needed as we restarting the goferd in background here https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L429 while registering